components/style/values/computed/mod.rs

```diff
@@ -586,3 +589,45 @@ impl ClipRectOrAuto {
 
 /// <color> | auto
 pub type ColorOrAuto = Either<Color, Auto>;
+
+/// The computed value of a CSS `url()`, resolved relative to the stylesheet URL.
+#[cfg(feature = "servo")]
+#[derive(Clone, Debug, HeapSizeOf, Serialize, Deserialize, PartialEq)]
+pub enum ComputedUrl {
```

Comment:
We could move this to another module to avoid the `cfg`'s, but that's probably doable as a followup.